JustLog/Classes/LogstashDestination.swift

```diff
@@ -40,7 +56,9 @@ public class LogstashDestination: BaseDestination  {
     
     public func cancelSending() {
         self.logDispatchQueue.cancelAllOperations()
-        self.socketManager.disconnect()
+        self.logsToShip = [Int : [String : Any]]()
```

Comment:
Since this func is public the access to `logsToShip` should be synchronized using the logDispatchQueue otherwise this will brake the thread safety